tfjs-backend-cpu/yarn.lock

```diff
@@ -313,6 +324,11 @@
   resolved "https://registry.yarnpkg.com/@types/seedrandom/-/seedrandom-2.4.27.tgz#9db563937dd86915f69092bc43259d2f48578e41"
   integrity sha1-nbVjk33YaRX2kJK8QyWdL0hXjkE=
 
+"@types/webgl-ext@0.0.30":
```

Comment:
Don't understand why "@types/webgl-ext@0.0.30" is needed for cpu and webgpu. @mattsoulanille

Reply:
`yarn why @types/webgl-ext` says that `@tensorflow/tfjs-core` depends on it. I think this is unintentional. Running `yarn remove @types/webgl-ext` in core and then running `yarn build-deps` and `yarn install` in backend-cpu seems to fix it.

Reply:
Thanks @mattsoulanille , I will close this change.